kotlin/feature-layer-selection/src/main/java/com/esri/arcgisruntime/sample/featurelayerselection/MainActivity.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.featurelayerselection
+
+import android.graphics.Color
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.MotionEvent
+import android.widget.Toast
+import com.esri.arcgisruntime.data.QueryParameters
+import com.esri.arcgisruntime.data.ServiceFeatureTable
+import com.esri.arcgisruntime.geometry.Envelope
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.layers.FeatureLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import kotlinx.android.synthetic.main.activity_main.*
+import java.lang.Exception
+import kotlin.math.roundToInt
+
+class MainActivity : AppCompatActivity() {
+
+  private val TAG: String = MainActivity::class.java.simpleName
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create service feature table and a feature layer from it
+    val serviceFeatureTable = ServiceFeatureTable(getString(R.string.gdp_per_capita_url))
+    val featureLayer = FeatureLayer(serviceFeatureTable)
+
+    // create a map with the streets base map type
+    val streetsMap = ArcGISMap(Basemap.createStreets()).apply {
+      // set an initial view point
+      initialViewpoint = Viewpoint(Envelope(
+        -1131596.019761, 3893114.069099, 3926705.982140, 7977912.461790,
+        SpatialReferences.getWebMercator()))
+      // add the feature layer to the map's operational layers
+      operationalLayers.add(featureLayer)
+    }
+
+    mapView.let {
+      // set the map to be displayed in the layout's map view
+      it.map = streetsMap
+      // give any item selected on the map view a red selection halo
+      it.selectionProperties.color = Color.RED
+
```

Comment:
maybe remove whitespace here and 2 lines below to match compactness of the rest of the sample